xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/v3/service/OkexTradeService.java

```diff
@@ -151,6 +151,11 @@ public OpenOrders getOpenOrders(OpenOrdersParams params) throws IOException {
     return new OpenOrders(all.stream().map(OkexAdaptersV3::convert).collect(Collectors.toList()));
   }
 
+  public OkexOpenOrder getOrdersDetails(String orderId, CurrencyPair pair) throws IOException {
+    final String instrument = OkexAdaptersV3.toSpotInstrument(pair);
+    return getSpotOrderDetails(orderId, instrument);
+  }
```

Comment:
this should be in the `*Raw` class because its not part o the generic API

Reply:
Done